sys-apps/dbus/dbus-1.12.6.recipe

```diff
@@ -0,0 +1,119 @@
+SUMMARY="D-Bus provides simple way for applications to talk to one another"
+DESCRIPTION="NOTE: D-Bus should not be used in native Haiku apps, but is only \
+here because some ported software requires it.
+
+D-Bus supplies both a system daemon (for events such as \"\
+new hardware device added\" or \"printer queue changed\") and a per-user\
+-login-session daemon (for general IPC needs among user applications). \
+Also, the message bus is built on top of a general one-to-one message passing \
+framework, which can be used by any two apps to communicate directly (without \
+going through the message bus daemon). Currently the communicating \
+applications are on one computer, or through unencrypted TCP/IP suitable for \
+use behind a firewall with shared NFS home directories."
+HOMEPAGE="http://freedesktop.org/wiki/Software/dbus"
```

Comment:
```diff
-HOMEPAGE="http://freedesktop.org/wiki/Software/dbus"
+HOMEPAGE="https://freedesktop.org/wiki/Software/dbus/"
```
(hint: there are 2 changes: the switch from `http` to `https` and the slash at the end :)